index.js

```diff
@@ -1,34 +1,23 @@
 import {
-    listCompanies,
     companyFrequency,
+    listCompanies,
```

Comment:
Let's list these to match the way they are called.
Can you move list Companies back to the top please?

Reply:
sure thing, boss. That's why you pay me the big bucks

Reply:
bingo bango. take your wife out somewhere nice. on me